docs/ceph-csi-upgrade.md

```diff
@@ -2,7 +2,7 @@
 
 - [Ceph-csi Upgrade](#ceph-csi-upgrade)
   - [Pre-upgrade considerations](#pre-upgrade-considerations)
-    - [snapshot-controller and snapshot Beta CRD](#snapshot-controller-and-snapshot-beta-crd)
+    - [snapshot-controller and snapshot Beta CRD](#snapshot-controller)
```

Comment:
Should this be:
`s/[snapshot-controller and snapshot Beta CRD]/[snapshot-controller and snapshot CRD]/` ?

Reply:
yeah.. correcting it.

Reply:
@pkalever addressed .. thanks. PTAL

Reply:
Is this not fixed yet? I still see beta CRD